beam/docs/zebra_printing.md

```diff
@@ -0,0 +1,178 @@
+# Zebra Printing
+
+To create a Zebra print format, you need the following documents:
+- A ZPL Print Format made against Handling Units that uses the available Jinja utility functions to generate ZPL code.
+- A document Print Format that uses the free Labelary API to convert the above ZPL code and generate a preview of the print output for the linked document.
+
+### ZPL Code Generation
+
+Currently, only three types of printable ZPL data can be generated with utilties within Beam:
+- `Text`
+- `Barcode`
+- `Label`
+
+Beam uses the [py-zebra-zpl](https://github.com/mtking2/py-zebra-zpl) library to generate the above types, as it provides a basic interface to create ZPL code using Python objects. Please refer to the library's documentation for more information on how to use it.
+
+In addition, Beam exposes the following Jinja functions to be used within a Print Format:
+
+---
+
+#### `barcode128`
+
+Generate a [Code 128](https://en.wikipedia.org/wiki/Code_128) barcode image. It takes the following arguments:
+
+- `barcode_text`: The text to be encoded in the barcode. Required.
+
+##### Example
+```jinja
+{{ barcode128('123456') }}
```

Comment:
Can we change the literal here to something more instructive, like `doc.barcodes[0].barcode`